### ISSUE TITLE
Create collection: gas estimate is a fraction of what's showing up in Metamask

### ISSUE THREAD

**Issue by c6y** [cryptovoxels/issues#121]
**Describe the bug**
Gas cost estimate when creating a collection will diverge wildly from what is showing up on Metamask.

**To Reproduce**
1. Go to 'Create a collection'
2. Click on 'Get a gas estimate'
3. > "Deploy cost: Eth 0.05649117 ($209.897); Gwei: 112.514000829"
4. Later on "Set and Deploy"
5. Click on 'Upload And Deploy'
6. Metamask > "www.cryptovoxels.com suggested gas fee $4,128.70 1.11211ETH Max fee: 1.550955ETH"

**Expected behavior**
Gas estimate should match or be close to final cost shown in Metamask

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser: Chrome
 - Version 92.0.4515.159

**Additional context**
Balance on wallet is 2.6 ETH

**Comment by Benjythebee** [comment]
Good catch, it was a mistake in the Math, should be fixed in a version higher than 8.2.1
